### PR TITLE
Feature/124

### LINKED ISSUE #124: Widget configuration not working
The config parameter of a widget is supposed to be passed as a prop to the rendering component.

So in

```
{
id: 'canvasExample',
name: 'My Canvas Exmaple',
component: 'canvas',
panelName: 'centralPanel',
enableClose: true,
enableRename: true,
enableDrag: true,
status: WidgetStatus.ACTIVE,
config: {
  data: getProxyInstances(),
},
}
```

data is the prop.
At the moment setting a config element causes the widget factory to break.

### PR DESCRIPTION
**Pull request by afonsobspinto**
closes #124

### CODE REVIEW

File: geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/SimpleInstancesExample.js [hunk 1]
```diff
@@ -66,8 +66,8 @@ class SimpleInstancesExample extends Component {
         reset: false,
         autorotate: false,
         wireframe: false,
-        showModel: false
       },
+      showModel: false
```
Comment (filippomc) [inline geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/SimpleInstancesExample.js:70]: is this change related to the widget issue?

File: geppetto.js/geppetto-client/js/common/layout/utils.ts [hunk 2]
```diff
@@ -13,9 +13,10 @@ export function widget2Node (configuration: Widget) {
     type: "tab",
     enableRename: false,
     enableClose,
+    panelName,
```
Comment (filippomc) [inline geppetto.js/geppetto-client/js/common/layout/utils.ts:16]: why is this needed?

File: geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/SimpleInstancesExample.js (hunk 1 above)
Comment (afonsobspinto) [reply geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/SimpleInstancesExample.js:70]: No

File: geppetto.js/geppetto-client/js/common/layout/utils.ts (hunk 2 above)
Comment (afonsobspinto) [reply geppetto.js/geppetto-client/js/common/layout/utils.ts:16]: It was being destructed but not used.
I assumed it was a mistake so I added it, but let me know if the mistake was me adding it :stuck_out_tongue:

Comment (filippomc) [reply geppetto.js/geppetto-client/js/common/layout/utils.ts:16]: it doesn't hurt, let's keep it

**Review by filippomc — APPROVED** [review]
lgtm
